### PR TITLE
Some further suggestions following the Sydney workshop

### PR DESCRIPTION
**Pull request by steveped**
Following the updates suggested by @hdashnow here are some further suggestions. Feel free to ignore or incorporate as you see fit.

(I've never forked or committed like this before, so let me know if I've done it wrong)

In brief:
- Vectors were more clearly introduced as containing all the same type of data.
- Subsetting was introduced for vectors first. This question spontaneously arose from the group at this point in the workflow
- Data frames are introduced as a collection of vectors
- Subsetting data frames was also introduced using the $ symbol, and treating each column as a variable

### CODE REVIEW

**Comment by hdashnow** [comment]
Hey @steveped,

I don't know about this repository, but many just ask you to submit .Rmd (or .md as appropriate) and not include the .html. That tends to avoid merge conflicts. The lesson maintainers then generate html at regular intervals.

**Comment by steveped** [comment]
Ah. Sorry...

________________________________

File: 02-starting-with-data.Rmd [hunk 1]
```diff
@@ -86,7 +86,8 @@ function `head()`:
 head(surveys)
 ```
 
-Let's now check the __str__ucture of this `data.frame` in more details with the
+An important feature of a `data.frame`is that each column is a vector, with the same type of data.
```
Comment (fmichonneau) [inline 02-starting-with-data.Rmd]: missing space after `data.frame`

**Comment by fmichonneau** [comment]
Hi @steveped,

  Thanks for the updates! As Harriet mentioned, it's best to only commit the Rmd files to avoid merge conflicts in the rendered lessons. Either create a new pull request to only include the Rmd files or delete the html files in this pull request, and I'll merge your changes.

  Cheers!

**Comment by fmichonneau** [comment]
hi @steveped,

I just wanted to follow up to see if you needed help with updating your pull request.

Thanks!

**Pull request closed without merging**